### DISCUSSION TITLE
Question about the large CI of the estimation

### DISCUSSION THREAD

**Discussion by YikeWangUCD** [JanDitzen/xtbreak#15] [Q&A]
Dear Jan

Hope you are well.

When I implement the xtbreak test for a time series(Hypothesis 1, s=1) and I find the supF statistics is very large (2300 for 1% critical value 12.72). Then I conduct the xtbreak estimate and I find the 95% CI for the estimation of SBD(structural break point) is very large (eg：2002m9 has CI from 1759m3 to 2246m3). I also tries test within the unknown points(Hypothesis 3) and estimate the SBD but the CI fro date is also very large. Would you like to explain this for me? It helps my research a lot. Super thanks!!

Kinds
Yike

**Comment by JanDitzen** [comment]
Hi,
thanks for your interest in xtbreak.

What kind of data do you use? In my experience the supF should not be that large. Did you also try different variance estimator?

Best,

Jan

**Reply by YikeWangUCD** [reply]
Dear Jan,
I hope you are well. 
I added more information two weeks ago. Would you like to help me solve the problem? It will helps a lot in my paper. Thank you in advance!!
Have a nice day:)

Kinds Yike

**Comment by YikeWangUCD** [comment]
Dear Jan

I'm sorry for the late reply as I miss your email before.

I used a kind of asset return time series data, as shown in figure1, that looks like white noise. When performing a stationary test (augment the Dickey fuller test), we cannot reject the null hypothesis, we can only accept the alternative hypothesis that the time series is stationary. But we notice that the time series fluctuates more than usual from 300-400 on the X-axis. When we divide the time series into 3 equal lengths, we find that the statistics for the 3 time periods are different:
# mean1=-0.036436, mean2=-0.024131, mean2=0.062130
# variance1=0.568122, variance2=1.439921, variance2=0.996324
However, further stationarity tests show that all three cycles are stationary. By the way, the estimation of SBD of this series is 362.
***@***.***
Figure 1: plot for the timeseries

Second, I tried different variance estimators. Different variance estimators seem to produce large estimation confidentiality intervals. SupF statistics are different for different variance estimators, but I think the key is to do with the type of data I apply. Some time series may produce normal SupF, while others will not. In addition, two different variance estimators for a same timeseries may produce two different results: one rejects the null hypothesis and the other rejects the null hypothesis.

Have a nice day😊 Thank you in advance!

Kind regards
Yike


发件人: Jan Ditzen ***@***.***>
日期: 星期二, 2024年4月2日 15:06
收件人: JanDitzen/xtbreak ***@***.***>
抄送: YikeWangUCD ***@***.***>, Author ***@***.***>
主题: Re: [JanDitzen/xtbreak] Question about the large CI of the estimation (Discussion #15)

Hi,
thanks for your interest in xtbreak.

What kind of data do you use? In my experience the supF should not be that large. Did you also try different variance estimator?

Best,

Jan

**Comment by JanDitzen** [comment]
Hi, 
I believe there is no easy answer to this. How large are the differences in the estimated coefficients for the different segments? If they are small, KI can become much larger.
Best,
Jan
